src/components/Popup.js

```diff
@@ -22,6 +22,14 @@ export default class Popup {
     }
   };
 
+  renderLoadingMessage(data) {
+    this._saveButton.textContent = data;
+  }
+
```

Comment:
This class renders any popup, it should not know anything about the submit button. You should implement the methods for rendering loading in PopupWithForm and PopupWithConfirmation classes